modules/serverless-ossm-v2x.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * serverless/networking/serverless-ossm.adoc
+
+[id="serverless-ossm-v2x_{context}"]
+= Using {ProductShortName} 2.x with {ServerlessProductName}
+
+.Procedure
+
+. Add the `default` namespace to the xref:../../service_mesh/v1x/installing-ossm.adoc#ossm-member-roll-create_installing-ossm[ServiceMeshMemberRoll] as a member:
++
+[source,yaml]
+----
+apiVersion: maistra.io/v2
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members:
+    - default
+  version: v2.0
```

Comment:
@nak3 can you advise here please

Reply:
Yes, we don't need this line (`version: v2.0`). Let's remove it.

Reply:
` version: v2.0` is necessary for `ServiceMeshControlPlane` not `ServiceMeshMemberRoll`. And we don't document about deploying `ServiceMeshControlPlane` because it is ServiceMesh's doc so we can roll bacck to single file as Chris said.